src/accountant_skel.rs

```diff
@@ -132,6 +132,40 @@ impl AccountantSkel {
         Ok(l)
     }
 
+    fn process_entry_list_into_blobs(
+        list: &Vec<Entry>,
+        blob_recycler: &packet::BlobRecycler,
+        q: &mut VecDeque<SharedBlob>,
+    ) {
+        let mut start = 0;
+        let mut end = 0;
+        while start < list.len() {
+            let mut total = 0;
+            for i in &list[start..] {
+                total += size_of::<Event>() * i.events.len();
+                total += size_of::<Entry>();
+                if total >= BLOB_SIZE {
+                    break;
+                }
+                end += 1;
+            }
+            // See that we made progress and a single
+            // vec of Events wasn't too big for a single packet
+            assert!(end > start);
+            let b = blob_recycler.allocate();
+            let pos = {
+                let mut bd = b.write().unwrap();
+                let mut out = Cursor::new(bd.data_mut());
+                serialize_into(&mut out, &list[start..end]).expect("failed to serialize output");
+                out.position() as usize
+            };
+            assert!(pos < BLOB_SIZE);
```

Comment:
Just confirming, `pos` being equal to `BLOB_SIZE` indicates a programatic error?